test/3x/torch/test_autotune.py

```diff
@@ -172,8 +177,14 @@ def eval_perf_fn(model) -> float:
             },
         ]
 
+        evaluator = _create_evaluator_for_eval_fns(eval_fns)
```

Comment:
It seems like when using multiple `eval_fns`, the `eval_args` is no longer supported.
Others LGTM.

Reply:
Yes, the `autotune` API only allows a single evaluation function (possibly paired using `eval_args`). In this case, I've wrap multiple evaluation functions into a single function to minimize change in the unit tests.